packages/opentelemetry-exporter-collector/src/platform/node/util.ts

```diff
@@ -49,6 +49,12 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
   };
 
   const request = parsedUrl.protocol === 'http:' ? http.request : https.request;
+  const Agent = parsedUrl.protocol === 'http:' ? http.Agent : https.Agent;
+  if (collector.keepAlive) {
+    const keepAliveAgent = new Agent({ keepAlive: true });
```

Comment:
I think we should allow to modify every options of the agent from the collector options.